libraries/common/io/error.effekt

```diff
@@ -366,8 +366,106 @@ def fromNumber(number: Int): IOError = number match {
 
 namespace internal {
 
+  extern llvm """
+    declare %Int @c_error_number(%Int)
+  """
+
+  extern jsNode """
+    const os = require('os');
```

Comment:
Same, 'os' is only available in node

Reply:
We could also use this:
```suggestion
    const os = require('node:os');
```
for consistency.